Sources/PointFree/AtomFeed.swift

```diff
@@ -8,6 +8,11 @@ let atomFeedResponse =
   writeStatus(.ok)
     >=> respond(pointFreeFeed, contentType: .application(.atom))
 
+let episodesRssMiddleware: Middleware<StatusLineOpen, ResponseEnded, SubscriberState, Data> =
```

Comment:
I also added a feed endpoint for all episodes, but it only shows trailers for the subscriber-only episodes.

I think we might actually be able to replace our current `feed/atom.xml` with this. we could prob just 301 it